zegami_sdk/collection.py

```diff
@@ -1289,3 +1289,8 @@ def _parse_source(self, source) -> Source:  # noqa: C901
                 'associated with this collection'.format(source.name))
 
         return s
+    
+    def add_source(self, payload):
```

Comment:
Better imo to have separate arguments for name and root_imageset_id. Should provide a docstring for this fn.